### PR TITLE
LPS-180002

### PR DESCRIPTION
**Pull request by ealonso**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by ealonso** [comment]
ci:forward

File: modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentFolderResourceTest.java [hunk 1]
```diff
@@ -41,18 +36,6 @@
 public class StructuredContentFolderResourceTest
 	extends BaseStructuredContentFolderResourceTestCase {
 
-	@Before
-	@Override
-	public void setUp() throws Exception {
-		super.setUp();
-
-		testDepotEntry = DepotEntryLocalServiceUtil.addDepotEntry(
```
Comment (Tim-Cao) [inline modules/apps/headless/headless-delivery/headless-delivery-test/src/testIntegration/java/com/liferay/headless/delivery/resource/v1_0/test/StructuredContentFolderResourceTest.java]: If don't create a new depot, then the test will get 3 folders

**Comment by ealonso** [comment]
@Tim-Cao thanks!! I've already fixed it.

**Comment by Tim-Cao** [comment]
Hi @ealonso, would you mind having a look at the failure? I cannot reproduce on my local. I don't know where did I make mistake.
It proved your previous commit was not the cause.

**Comment by ealonso** [comment]
Hey @Tim-Cao,

Don't worry about it, I'll take care.

**Comment by Tim-Cao** [comment]
@ealonso
I figured out the reason, the profileURL is generated after sign in via UI. However, ci won't sign in via UI when run integration tests.

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/12274

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**